frontend/components/App/Participants/index.tsx

```diff
@@ -120,21 +120,21 @@ const Participants: React.FC<Props> = ({ initialized, fid, toggleParticipants })
       pingParticipant();
       getApiParticipants();
 
-      setPingInterval(window.setInterval(pingParticipant, PING_TIMEOUT));
-      setGetParticipantsInterval(window.setInterval(getApiParticipants, PING_TIMEOUT));
+      pingInterval.current = window.setInterval(pingParticipant, PING_TIMEOUT);
+      getParticipantsInterval.current = window.setInterval(getApiParticipants, PING_TIMEOUT);
     } else {
-      window.clearInterval(pingInterval);
-      window.clearInterval(getParticipantsInterval);
+      window.clearInterval(pingInterval.current);
+      window.clearInterval(getParticipantsInterval.current);
 
       setTimeout(() => {
         // Each 1 second it will fetch users from Jitsi to show the sidebar and user count correctly
-        setGetParticipantsInterval(window.setInterval(getConferenceParticipants, 1000));
+        getParticipantsInterval.current = window.setInterval(getConferenceParticipants, 1000);
       }, PING_TIMEOUT);
     }
 
     return () => {
-      window.clearInterval(pingInterval);
-      window.clearInterval(getParticipantsInterval);
+      window.clearInterval(pingInterval.current);
+      window.clearInterval(getParticipantsInterval.current);
```

Comment:
Changed from state to ref because on unmount the ref is accesible, the state not.